### ISSUE TITLE
Problems with Apache/2.4.7 (Ubuntu 14.04)

### ISSUE THREAD

**Issue by droptablestar** [chdemko/apache-pandoc#4]
When trying to get this to work I was only able to have Apache serve HTML when I ran `sudo a2enmod actions`. I am able to return HTML when accessing only the `.md` extensions, not `.html`, `.pdf`, `.rtf`, `.odt` and `.raw` files listed in the README. It doesn't appear the style sheet is being used at all.

**Comment by chdemko** [comment]
Do you have enable the mod_rerwrite module?

**Comment by droptablestar** [comment]
Yes. I'm also wondering if the css file is getting injected into the generated HTML. This is the style tag that I see generated: <style type="text/css">code{white-space: pre;}</style>

**Comment by chdemko** [comment]
It seems there is an issue with rewriting.
